vyxal/elements.py

```diff
@@ -4133,10 +4133,62 @@ def roman_numeral(lhs, ctx):
         "IV",
```

Comment:
Clean up after yourself pls? (also it would be one line to fix #1178 while you're at it)